index.html

```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
         </ul>
       </div>
     </section>
+    <section class="works-container" id="works">
+      <div class="works-header-container header-area">
+        <div class="works-header">
+          <h2>My recent works</h2>
+          <div class="line"></div>
```

Comment:
- [OPTIONA] Well done with the design 🥇🥇 What do you think of adding a bit more space here according to the Figma template? 🤔✨ Also, can you please use `font-weight: 400` for  **_My recent works_** ? 😄

Your design | Given design
---| ---|
![image](https://github.com/Xecyc/My-Portfolio-Project/assets/106119105/94b0fe79-fd74-4b0a-822d-1a84503c4462) | ![image](https://github.com/Xecyc/My-Portfolio-Project/assets/106119105/715f8eb9-6d4e-416d-993f-5b0c93e47703)